src/assignments/mars-rover.rst

```diff
@@ -0,0 +1,137 @@
+Assignment #3: Mars Rover
+=========================
+This task is going to put your unit tests, modules, and exceptions knowledge to use
+by writing tests and classes for the Mars rover named Curiosity.
+
+.. figure:: figures/curiosity-rover-selfie.jpg
+   :alt: Image of Curiosity rover taken by the rover on Mars.
+
+   Selfie of the Curiosity on Mars.
```

Comment:
This caption looks weird. Can it be centered?

Reply:
It probably could, but that would require sphinx work.